server/aap/commands/export_legal_archive_to_archive.py

```diff
@@ -0,0 +1,155 @@
+            self._export_to_archive()
+        finally:
+            unlock(lock_name)
+
+        logger.info('Completed exporting {} desk legal archive content to archive.'.format(self.default_desk))
+
+    def _get_desk_id(self):
+        """Returns the ObjectID of the desk
+        """
+        logger.info('Fetching the ObjectID for the desk {}.'.format(self.default_desk))
+        query = {'name': self.default_desk}
+        req = ParsedRequest()
+        req.where = json.dumps(query)
+
+        desk_service = get_resource_service('desks')
+        desk_item = list(desk_service.get_from_mongo(req=req, lookup=None))
+        if not desk_item:
+            logger.error('Failed to find the ObjectID for the provided desk {}'.format(self.default_desk))
+            return None
+
+        desk_id = desk_item[0]['_id']
+        logger.info('ObjectID for the desk {} is {}.'.format(self.default_desk, desk_id))
+        return desk_id
+
+    def _export_to_archive(self):
+        """Export legal archive content to archive
+
+         Copy all legal archive content that belongs to the desk provided
+         into archive
+        """
+        logger.info('Exporting legal archive content to archive.')
+        items = list()
+        try:
+            desk_id = self._get_desk_id()
+            if not desk_id:
+                return
+
+            # from pprint import pprint
+            for item in self._get_items():
+                items.append(self._generate_archive_item(item, desk_id))
+                # pprint(item)
+                # input('Awaiting your response')
+
+            self._add_to_archived(items)
+        except Exception as e:
+            logging.exception('Failed to export legal archive content to archive. {}'.format(e))
+
+    def _get_items(self):
+        """Get items from the LegalArchive that belong to the COMMISSION desk
+
+        :return: list: list of legal archive content
+        """
+        logger.info('Fetching legal archive content for the {} desk'.format(self.default_desk))
+
+        query = {'task.desk': self.default_desk, 'type': 'text'}
+        req = ParsedRequest()
+        req.where = json.dumps(query)
+
+        legal_archive_service = get_resource_service(LEGAL_ARCHIVE_NAME)
+
+        legal_items = list(legal_archive_service.get_from_mongo(req=req, lookup=None))
+
+        if legal_items:
+            logger.info(
+                'Found {} items in the legal archive for the {} desk'.format(
+                    len(legal_items),
+                    self.default_desk)
+            )
+        else:
+            logger.warning('Failed to find any {} desk items in the legal archive'.format(self.default_desk))
+            legal_items = []
+
+        return legal_items
+
+    def _generate_archive_item(self, item, desk_id):
+        """Convert the legal archive content for the archive content\
+
+        :param dict item: legal archive item
+        :return dict: archive item converted from legal archive item
```

Comment:
Even though it is self evident I think it is good practice to add all parameters to the comments.